_posts/2019-02-01-dynamic-spawn-scheduler.md

```diff
@@ -48,9 +48,15 @@ There are a couple of prerequisites to use the dynamic spawn scheduler feature:
 
 If your setup satisfies the above, all you have to do is specify `--experimental_spawn_scheduler` on your next build. Just make sure to pass this new flag after all others.
 
+*WARNING: As the flag name implies, this feature is experimental. In particular, be aware that this could introduce correctness issues if the local and remote build environments aren't sufficiently similar to yield the same outputs for each action.*
```

Comment:
maybe link to https://docs.bazel.build/versions/master/backward-compatibility.html#at-a-glance

Reply:
Done.